assets/js/admin/marketplace-suggestions.js

```diff
@@ -0,0 +1,411 @@
+/* global marketplace_suggestions ajaxurl */
+( function( $, marketplace_suggestions ) {
+	$( function() {
+		if ( 'undefined' === typeof marketplace_suggestions ) {
+			return;
+		}
+
+		// Stand-in wcTracks.recordEvent in case tracks is not available (for any reason).
+		window.wcTracks = window.wcTracks || {};
+		window.wcTracks.recordEvent = window.wcTracks.recordEvent  || function() { };
+
+		// Tracks events sent in this file:
+		// - marketplace_suggestion_displayed
```

Comment:
💟thanks for this documentation right here, being able to easily locate where tracks are recorded is such a lovely thing.

Reply:
Let's all promise to keep it updated when things change 🤝